src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -76,24 +75,19 @@ public void worksFine() throws Exception {
         );
     }
 
-    // @todo #78:30min With #78 solved, HtWire stopped closing the connection
-    //  when send() is called and the following test stopped working for no
-    //  clear reason. Investigate the cause of this, fix it and unignore this.
-    @Ignore("see todo above")
+    // @todo #87:30m For now I can't find proper OOP alternative for waiting
+    //  (such in this case). Needs to create a new one.
```

Comment:
@aivinog1 Could you also describe what this alternative would need to have as opposed to the current implementation?

Reply:
Sure. I'll add my thoughts about that in this comment.

Reply:
Done.